consumer.py

```diff
@@ -50,13 +50,15 @@
 from confluent_kafka import Consumer
 from confluent_kafka import TopicPartition
 
+
 # We run all the modules so that their metrics and handlers get registered
 run_module("thoth.investigator.advise_justification.investigate_advise_justification")
 run_module("thoth.investigator.adviser_re_run.investigate_adviser_re_run")
 run_module("thoth.investigator.adviser_trigger.investigate_adviser_trigger")
 run_module("thoth.investigator.build_analysis_trigger.investigate_build_analysis_trigger")
 run_module("thoth.investigator.cve_provided.investigate_cve_provided")
 run_module("thoth.investigator.hash_mismatch.investigate_hash_mismatch")
+run_module("thoth.investigator.inspection_complete.investigate_inspection_complete")
```

Comment:
here and below in all changes should be `completed` and not `complete` to keep consistency in naming :)